docs/pre-existing-playbook-integration.md

```diff
@@ -0,0 +1,130 @@
+# Integrating Pre-existing Playbooks with APBs
+
+1. [Introduction](#introduction-to-playbook-integration)
+1. [Distinguishing Actions](#distinguishing-actions)
+    1. [Ansible 'when' Statement](#ansible-when-statement)
+1. [Pre-existing Playbook Installation](#pre-existing-playbook-installation)
+    1. [Ansible Galaxy](#ansible-galaxy)
+    1. [RPMs](#rpms)
+    1. [GitHub Source](#github-source)
+1. [Running the Pre-existing Playbooks](#running-the-pre-existing-playbooks)
+    1. [Shim Playbook](#shim-playbook)
+
+## Introduction to Playbook Separation
+APBs expect a specific API in the form of playbook names like
```

Comment:
I think API here is confusing. We expect playbooks named after OSB actions `provision` being one.